boolean_algebra/norgate.py

```diff
@@ -0,0 +1,49 @@
+""" A NOR Gate is a logic gate in boolean algebra which results to false(0)
+    if any of the input is 1, and True(1) if  both the inputs are 0.
+   Following is the truth table of an NOR Gate:
+   | Input 1 | Input 2 |  Output |
+   |      0      |     0      |      1      |
+   |      0      |     1      |      0      |
+   |      1      |     0      |      0      |
+   |      1      |     1      |      0      |
+"""
+"""Following is the code implementation of the NOR Gate"""
+
+
+def nor_gate(input_1: int, input_2: int) -> int:
+    """
+    >>> nor_gate(0, 0)
+    1
+    >>> nor_gate(0, 1)
+    0
+    >>> nor_gate(1, 0)
+    0
+    >>> nor_gate(1, 1)
+    0
+    >>> nor_gate(0.0, 0.0)
+    0
+    >>> nor_gate(0, -7)
+    0
```

Comment:
This doctest will fail...
```suggestion
    1
```

Reply:
On your computer, you can run these tests with:
`python3 -b doctest -v boolean_algebra/norgate.py`